bower.json

```diff
@@ -16,7 +15,10 @@
     "wct.conf.json"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0"
+    "polymer": "Polymer/polymer#^2.0.0",
+    "d2l-dropdown": "^6.6.2",
+    "d2l-menu": "^1.2.0",
+    "d2l-icons": "^5.1.2"
```

Comment:
Nit: general best practice is to order the deps alphabetically.  Minor thing especially with a list this short... just calling it out since it will probably be raised elsewhere in the future so good to get in the habit.  :)